### ISSUE TITLE
[BUG?] two events instead of one event sent by some devices

### ISSUE THREAD

**Issue by kevincaradant** [Koenkk/zigbee2mqtt#5022]
Hi

Maybe (again) it's something that I don't understand the utility but I saw these logs and I don't understand why motion sensor and switches (when I click on it) send two events.

For the motion sensor, it's the same at the same time.

For the switches, it's the same but one with the action and one without action.

On an other thread, I saw something like it was because I have `homeassistant: true` but I use HA with the mqtt broker (connected to mosquitto) 
It's seems to be the same topics (except the third event, so I can understand the third event, but not the first and second which are the same)  so I don't think so it's related to the HA option ?

```
zigbee2mqtt    | Zigbee2MQTT:info  2020-11-19 22:51:52: MQTT publish: topic 'zigbee2mqtt/switch_bed_bedroom', payload '{"action":"right","battery":100,"linkquality":120,"voltage":3205}'
zigbee2mqtt    | Zigbee2MQTT:info  2020-11-19 22:51:52: MQTT publish: topic 'zigbee2mqtt/switch_bed_bedroom', payload '{"action":"","battery":100,"linkquality":120,"voltage":3205}'
zigbee2mqtt    | Zigbee2MQTT:info  2020-11-19 22:51:52: MQTT publish: topic 'zigbee2mqtt/switch_bed_bedroom/action', payload 'right'
zigbee2mqtt    | Zigbee2MQTT:info  2020-11-19 22:51:58: MQTT publish: topic 'zigbee2mqtt/router_pc_desktop', payload '{"linkquality":117,"power":98.8,"state":"ON"}'
zigbee2mqtt    | Zigbee2MQTT:info  2020-11-19 22:52:00: MQTT publish: topic 'zigbee2mqtt/motion_bedroom', payload '{"battery":100,"illuminance":2,"illuminance_lux":2,"linkquality":129,"occupancy":true,"voltage":3025}'
zigbee2mqtt    | Zigbee2MQTT:info  2020-11-19 22:52:00: MQTT publish: topic 'zigbee2mqtt/motion_bedroom', payload '{"battery":100,"illuminance":2,"illuminance_lux":2,"linkquality":129,"occupancy":true,"voltage":3025}'
```

Thank you :)

**Comment by Koenkk** [comment]
This is expected, see https://github.com/Koenkk/zigbee2mqtt/issues/1155#issuecomment-467094619

Tldr; required for home assistant integration
